Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+      - CyberPen
+      - CigPackSyndicate
+      - WeaponRevolverPythonAP
+      - ClothingEyesGlassesOutlawGlasses
+      - SyndicatePersonalAI
+      - EnergySword
+      - WeaponPistolCobra
+      - EmpGrenade
+      - Stimpack
+      - ClusterGrenade
+      - EncryptionKeyBinary
+      - EnergyShield
+      - SyndieMiniBomb
+      - PenExplodingBox
+      - GrenadeShrapnel
+      - GrenadeIncendiary
+      - ExGrenade
+      - C4
+      - SmokeGrenade
+      - GrenadeFlashBang
+      - ClothingShoesBootsMagSyndie
+      - ClothingBackpackDuffelSyndicateEVABundle
+      - ClothingUniformJumpskirtOperative
+      - ClothingOuterVestWeb
+      - ClothingNeckScarfStripedSyndieGreen
+      - ClothingUniformJumpsuitOperative
+      - ClothingHandsGlovesBoxingRigged
+      - ScramImplanter
+      - StorageImplanter
+      - ClothingHeadHatOutlawHat
+      - ClothingBackpackDuffelSyndicateCostumeClown
+      - RubberStampSyndicate
+      - ClothingBackpackDuffelSyndicatePyjamaBundle
+      - ClothingBackpackDuffelSyndicateCarpSuit
+      - EnergyDagger
+      - RevolverCapGun
+      - BalloonSyn
+      - ClothingHeadHatCatEars
+      - RadioJammer
+      - JetpackBlackFilled
+      - SoapSyndie
+      - BoxEncryptionKeySyndie
+      - ClothingHeadsetAltSyndicate
+      - HappyHonkNukieSnacks
+      - NukeDiskFake
+      - LanternFlash
+      - CrateFunSyndicateSegway
+      - SlipocalypseClusterSoap
+      - CombatMedipen
+      - BriefcaseSyndieLobbyingBundleFilled
+      - MedkitCombatFilled
+      - Emag
+      - ClothingBackpackDuffelSyndicateBundleSamurai
+      - BorgModuleSyndicateWeapon
+      - ClothingOuterHardsuitSyndie
+      - ClothingOuterHardsuitSyndieMedic
+      - SyndiHypo 
+      - WeaponRifleAk
+      - WeaponRifleLecter
+      - WeaponPistolCHIMPUpgradeKit
+      - WeaponSniperHristov
+      - WeaponPistolN1984
+      - WeaponLauncherPirateCannon
+      - ClothingHeadHelmetSwatSyndicate
+      - ClothingShoesBootsWinterSyndicate
+      - ClothingBeltMilitaryWebbingMedFilled
+      - EnergyKatana
+      - PenCentcom
+      - FoodPizzaArnold
+      - WeaponShotgunEnforcer
+      - SadTromboneImplanter
+      - PinpointerUniversal
+      - BluespaceBeaker
+      - ClothingBackpackHolding
+      chance: 1
+      rarePrototypes:
+       - ClothingBackpackDuffelSyndicateFilledGrenadeLauncher
+       - StealthBox
+       - ClothingBackpackDuffelSyndicateFilledEmpGrenadeLauncher
+       - ClothingBackpackDuffelSyndicateFilledLMG
+       - ClothingBackpackDuffelSyndicateFilledShotgun
+       - ClothingBackpackDuffelSyndicateFilledSMG
+       - BriefcaseSyndieSniperBundleFilled
+       - ElectricalDisruptionKit
+       - ChemicalSynthesisKit
+       - ClothingOuterHardsuitSyndieElite
+       - EmpImplanter
+       - ClothingBackpackChameleonFill
+       - NocturineChemistryBottle
+       - ThievingGloves
+       - ScramImplanter
+       - ClothingOuterHardsuitJuggernaut
+       - WeaponPistolViper
+       - WhiteholeGrenade
```

Comment:
Opinions on making this common?  No explosion, just pushes stuff away.

Reply:
I mean i could but i was thinking of the effects it would have if you throught it outside and there is people in space... it could be a REAL shitter thing that can happen with it